scout/server/blueprints/cases/controllers.py

```diff
@@ -219,6 +222,55 @@ def case_report_content(store, institute_obj, case_obj):
     return data
 
 
+def coverage_report_contents(store, institute_obj, case_obj, base_url):
+    """Posts a request to chanjo-report and capture the body of the returned response to include it in case report
+
+    Args:
+        store(adapter.MongoAdapter)
+        institute_obj(models.Institute)
+        case_obj(models.Case)
+        base_url(str): base url of server
+
+    Returns:
+        coverage_data(str): string rendering of the content between <body </body> tags of a coverage report
+    """
+
+    request_data = {}
+    # extract sample ids from case_obj and add them to the post request object:
+    request_data['sample_id'] = [ ind['individual_id'] for ind in case_obj['individuals'] ]
+
+    # extract default panel names and default genes from case_obj and add them to the post request object
+    distinct_genes = set()
```

Comment:
This is not used? If not used, remove

Reply:
@northwestwitch what are you thinking of this?

Reply:
Genes don't need to be passed to chanjo-report, just panels, so it's safe to remove the commented code